### PR TITLE
ice 3.7.6

### PR DESCRIPTION
**Pull request by externl**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
The build fails on Mojave:
```
src/IceSSL/SecureTransportTransceiverI.cpp:95:14: error: use of undeclared identifier 'errSecCertificateNameNotAllowed'
        case errSecCertificateNameNotAllowed:
             ^
src/IceSSL/SecureTransportTransceiverI.cpp:100:14: error: use of undeclared identifier 'errSecCertificatePolicyNotAllowed'
        case errSecCertificatePolicyNotAllowed:
             ^
src/IceSSL/SecureTransportTransceiverI.cpp:119:14: error: use of undeclared identifier 'errSecCertificateValidityPeriodTooLong'
        case errSecCertificateValidityPeriodTooLong:
             ^
```

**Comment by carlocab** [comment]
> The build fails on Mojave:
> 
> ```
> src/IceSSL/SecureTransportTransceiverI.cpp:95:14: error: use of undeclared identifier 'errSecCertificateNameNotAllowed'
>         case errSecCertificateNameNotAllowed:
>              ^
> src/IceSSL/SecureTransportTransceiverI.cpp:100:14: error: use of undeclared identifier 'errSecCertificatePolicyNotAllowed'
>         case errSecCertificatePolicyNotAllowed:
>              ^
> src/IceSSL/SecureTransportTransceiverI.cpp:119:14: error: use of undeclared identifier 'errSecCertificateValidityPeriodTooLong'
>         case errSecCertificateValidityPeriodTooLong:
>              ^
> ```

Would you know if this supports this version of macOS? Is it just another Xcode issue? Mojave caps out at 11.3 (I think).

**Comment by externl** [comment]
Yeah, Mojave (10.14) is no longer supported. I'll update the formula.

File: Formula/ice.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Ice < Formula
 
   depends_on "lmdb"
   depends_on "mcpp"
+  depends_on macos: :catalina
```
Comment (carlocab) [inline Formula/ice.rb:22]: Did you run `brew audit` on this? It might want you to put this line before `mcpp`.

Comment (externl) [reply Formula/ice.rb:22]: I did, it didn't complain.

Comment (externl) [reply Formula/ice.rb:22]: hmmm.. I ran it again and it failed :/. Will fix

**Review by carlocab — APPROVED** [review]
Thanks, @externl.

**Pull request closed without merging**